docs/iam-policy-example.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "elasticfilesystem:CreateAccessPoint",
+        "elasticfilesystem:DeleteAccessPoint",
+        "elasticfilesystem:DescribeFileSystems",
```

Comment:
what about permission to mount access points to delete their contents? Since the controller is the one creating the access point i guess it doesn't need it?

Reply:
Not required for the controller as these permissions are used to talk to EFS APIs. IIRC, the mount permissions are defined on the file systems and exist as file system policies to limit access to file system.

Reply:
got it.

the default for a given fs is to allow all mountsso if user has some kind of custom restriction on filesystem they need to edit that policy themself https://docs.aws.amazon.com/efs/latest/ug/iam-access-control-nfs-efs.html